core/metamod_oslink.h

```diff
@@ -100,10 +100,6 @@ bool GetFileOfAddress(void *pAddr, char *buffer, size_t maxlength);
 	typedef unsigned __int32	uint32_t;
 #elif defined __GNUC__
 #include <stdint.h>
-#if !__GLIBC_HAVE_LONG_LONG
```

Comment:
I don't remember why this change was made, but it should be safe to require modern toolchains. I think we already do anyway for recentish FastDelegate changes (variadic templates).